app/_config/filters.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
```

Comment:
For each docType (e.g. `entities`) we might have a number of filters.
Each filter is defined by a (`field', `[options]`) tuple, and each options has a `value` and `name`.
 
I think current configuration doesn't allow that...